packages/eslint-plugin-pf-codemods/lib/rules/nav-list-variant.js

```diff
@@ -0,0 +1,40 @@
+const { getPackageImports } = require('../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/4225
+module.exports = {
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+        .filter(specifier => ['NavList', 'Nav'].includes(specifier.imported.name));
+    const navImport = imports.find(imp => imp.imported.name === 'Nav');
+    const NavListImport = imports.find(imp => imp.imported.name === 'NavList');
+
+    return !navImport || !NavListImport ? {} : {
+      JSXElement(node) {
+        if (NavListImport.local.name === node.openingElement.name.name) {
+          const hasNavParent = node.parent
+              && node.parent.openingElement.name.name === navImport.local.name
+              && node.parent.children.filter(child => child.type === 'JSXElement').length === 1;
+          const variantAttr = node.openingElement.attributes.find(attribute => {
+              return attribute.name.name === 'variant'
+          });
+
+          if (variantAttr) {
+            const variantVal = '"horizontal" || "default" || "tertiary"';
```

Comment:
Can you change this to `const variantVal = context.getSourceCode().getText(variantAttr) || '"horizontal" | "default" | "tertiary"';`?

Reply:
I removed the '|| '"horizontal" | "default" | "tertiary"' as none of the test cases are now using that value.